.circleci/config.yml

```
@@ -0,0 +1,72 @@
+version: 2
+
+jobs:
+  build:
+
+    # We use the machine executor, i.e. a VM, not a container
+    machine: true
+
+    steps:
+      - checkout
+
+      - run:
+          name: Ensure we have a clean docker environment
+          command: |
+            docker images "fundocker/cookiecutter"
+            test -z $(docker images -q fundocker/cookiecutter)
+
+      - run:
+          name: Build image
+          command: |
+              bin/build
+
+      - run:
+          name: Check built image
+          command: |
+            docker images "fundocker/cookiecutter"
+            ! test -z $(docker images -q fundocker/cookiecutter)
+            echo "Cookiecutter image exists"
+
+  publish:
+
+    # We use the machine executor, i.e. a VM, not a container
+    machine: true
```

Same here, use `docker_laying_caching`

```suggestion
    machine:
      docker_layer_caching: true
```